src/env.js

```diff
@@ -0,0 +1,24 @@
+const DEFAULT_INSTANCE_ID = 1;
+
+if (process.env.NODE_ENV !== 'production') {
+  process.env.CONSOLE_LOGGING = 'true';
+}
+
+process.env.MIN_LOG_LEVEL = 'info';
+process.env.LOG_TIMESTAMP = 'true';
+process.env.LOG_ENVIRONMENT = 'true';
+process.env.LOG_LEVEL = 'true';
+process.env.LOG_REQID = 'true';
+
+if (!process.env.HOSTNAME) {
+  process.env.HOSTNAME = process.env.USER;
```

Comment:
That's what I took from the old project, I haven't changed this file (except for adding new envs). This value (``process.env.HOSTNAME`` is then set as an ``instanceId``). If it is not set, ``undefined`` will be printed out all the time